hyperspy/signal.py

```diff
@@ -1865,7 +1865,8 @@ def __call__(self, axes_manager=None):
         return np.atleast_1d(
             self.data.__getitem__(axes_manager._getitem_tuple))
 
-    def plot(self, navigator="auto", axes_manager=None, **kwargs):
+    def plot(self, navigator="auto", axes_manager=None, 
+            plot_markers=False, **kwargs):
```

Comment:
I'm thinking `BASE_PLOT_DOCSTRING` might be the right place? Since this doesn't go to the `imshow`?